docker_all-in.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+echo -e "\n[ Cleaning docker ]\n"
+
+echo "Stoping docker"                                
+docker-compose stop                                  
+                                                     
+echo "Removing Compose containers"                   
+docker-compose rm --all -f                           
+                                                     
+docker-compose ps                                    
+                                                     
+docker ps                                            
+                                                     
+echo "Removing Docker containers, images and volumes"
+                                                     
+docker stop $(docker ps -a -q)                       
+docker rm -f $(docker ps -a -q)                      
+docker rmi -f $(docker images -q)                    
+docker volume rm -f $(docker volume ls  -q)         
+
+echo -e "\n[ Building Docker ]\n"
+
+docker-compose build
+
+echo -e "\n[ Starting Docker Compose ]\n"
+docker-compose up -d
+
+echo -e "\n[ Updating Docker Database ]\n"
+docker exec dpg_api python manage.py makemigrations authentication
+docker exec dpg_api python manage.py migrate --noinput      
+
+echo -e "\n[ Creating superuser ]\n"
+docker exec -it dpg_api python manage.py shell -c "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', 'admin@example.com', 'Admin123!')"        
```

Comment:
Its not good practice use `docker exec` here. 
As for me will be good use here django manage command. [Docs](https://docs.djangoproject.com/en/2.2/howto/custom-management-commands/#module-django.core.management).

Reply:
@klekhaav Are you agree with me about commands?

Reply:
it was for dev only to easily add superuser)))